addons/controllers/cpi/vspherecpiconfig_utils.go

```diff
@@ -133,27 +127,9 @@ func (r *VSphereCPIConfigReconciler) mapCPIConfigToDataValuesNonParavirtual( //
 		d.NoProxy = cluster.Annotations[pkgtypes.NoProxyConfigAnnotation]
 	}
 
-	// derive nsxt related configs from cluster variable
-	d.Nsxt.PodRoutingEnabled = r.tryParseClusterVariableBool(cluster, NsxtPodRoutingEnabledVarName)
```

Comment:
are we no longer deriving those nsxt configs from cluster variables?

Reply:
Yes, they are not related to topology and will never be Cluster variables

Reply:
thanks. I see now. those nsxt fields for CPI package are only coming from CPIConfig resource that are set by the addon